src/main/java/ru/hogwarts/school/model/Student.java

```diff
@@ -0,0 +1,44 @@
+package ru.hogwarts.school.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+@Entity
+public class Student {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String name;
+    private int age;
+
```

Comment:
В сущнсотях должен быть конструктор без параметров
```suggestion
    public Student() {
    }
```